code/__DEFINES/dcs/signals/signals_radio.dm

```diff
@@ -0,0 +1,2 @@
+/// Called when a radio jammer pulses to disable radio.
+#define COMSIG_RADIO_JAMMED "radio_jammed"
```

Comment:
Probably doesn't need to be in a separate file?

Reply:
There wasn't really any other file to put this in

Reply:
> There wasn't really any other file to put this in

COMSIG_ATOM_RADIO_JAMMED?